### DISCUSSION TITLE
Is SSL passthrough supported?

### DISCUSSION THREAD

**Discussion by itaydj** [nginx/kubernetes-ingress#2108] [Q&A] [answered]
I found this old issue saying it is not supported and was wandering if thats still the case

https://github.com/nginxinc/kubernetes-ingress/issues/492

Thanks in advance

**Comment by brianehlert** [comment, marked as answer]
the TransportServer resource handles layer 4 TCP/UDP and is commonly used for generic passthrough use cases.
https://docs.nginx.com/nginx-ingress-controller/configuration/transportserver-resource/

Since no layer 7 capabilities around inspection or manipulation will be possible.

**Reply by itaydj** [reply]
Thank you very much for the quick reply!

**Comment by jjngx** [comment]
Closing - issue solved
